src/mui/input/ImageInput.js

```diff
@@ -28,6 +28,15 @@ export class ImageInput extends Component {
         this.state = { files };
     }
 
+    componentWillReceiveProps(nextProps) {
```

Comment:
Note: as `componentWillReceiveProps` is not called at initial mount, we need both it and constructor initialization.